content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,111 @@
 ---
 title: Istio 1.1
-publishdate: 2019-03-01
+publishdate: 2019-03-12
 icon: notes
 ---
 
-TODO announcement
+We're proud to release Istio 1.1! It's been a while since 1.0 and we've done some significant improvements to make
+the overall product more reliable, more efficient, and more scalable.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.1.
 
 {{< relnote_links >}}
 
-## Policies and Telemetry
+## Upgrades
+
+- **Helm Changes**. 
+TBD: need full content & links
+
+- **TBD: TITLE**, Control plane and data plane requested resource changes
+TBD: need full content & links
+
+## Traffic management
+
+- **Allow All Outbound Traffic**. Allow all outbound traffic by default now. 
+To enforce no outbound traffic at mesh wide, configure `global.outboundTrafficPolicy.mode` to `REGISTRY_ONLY`
+TBD: presumbly in Helm?
+
+- **Egress Gateway Disabled By Default**.- Egress gateway is disabled by default now.
+To enable it, configure `gateways.istio-egressgateway.enabled` to `true`
+TBD: presumbly in Helm?
+
+- **Simpler Service Entry Resources**. Reduced the need to bind virtual services to service entry resources greatly simplifying the resources.
 
-- **Kiali**. The Service Graph addon has been [deprecated](https://github.com/istio/istio/issues/9066) in favor of [Kiali](https://www.kiali.io). See the [Kiali Task](/docs/tasks/telemetry/kiali/) for more details about Kiali.
+- TBD: note about sidecar and export_to API.
 
 ## Security
```

Comment:
Here are the security features added in 1.1 release. Aggregated notes from @wattli @myidpt @wenchenglu

**Identity provisioning through SDS** provides stronger security with on-node key generation and dynamic certificate rotation without restarting Envoy. Refer to guide [Provisioning Identity through SDS](/docs/tasks/security/auth-sds).
**Authorization for TCP services**  supports authorization for TCP services in addition to HTTP and gRPC services. Refer to guide [Authorization for TCP Services](/docs/tasks/security/authz-tcp).
**Authorization for end user groups** allows authorization based on `groups` claim or any list-typed claims in JWT. Refer to guide [Authorization for groups and list claims](/docs/tasks/security/rbac-groups/).
**End user authentication with per-path requirements** allows you enable or disable JWT authentication based on request path. Refer to guide [End-user authentication with per-path requirements](/docs/tasks/security/authn-policy/#end-user-authentication-with-per-path-requirements).
**External certificate management on ingress gateway controller** dynamically loads and rotates external certificates
Stronger security with SDS-based certificate/key loading. Refer to guide [Securing Gateways with HTTPS Using Secret Discovery Service](/docs/tasks/traffic-management/secure-ingress/sds).
**Vault PKI integration** provides stronger security with Vault-protected signing keys and facilitates integration with existing Vault PKIs. Refer to guide [Istio Vault CA Integration](/docs/tasks/security/vault-ca).
**Customized (non “cluster.local”) trust domains** supports organization or cluster specific trust domains in the identities.

Reply:
Thanks, I've integrated that.